wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

```diff
@@ -0,0 +1,83 @@
+
+## Syncing the fork
+
+In order to make changes to a repository located on GitHub, a potential contributor needs to obtain a controlled copy of it called a *fork*. When you create your fork of the `osu-wiki` repo, you take a snapshot of its contents at this very moment. To make a meaningful contribution, **always sync your fork** before making a set of changes—this can be done directly from GitHub:
+
+1. Go to your fork of the `osu-wiki` repo.
+2. Select the `master` branch from the dropdown.
+3. Click `Fetch upstream`, and select `Fetch and merge`.
+
+![](img/update-branch.png "Updating the outdated branch")
+
+Now your branch is up-to-date with the original repository.
+
+---
+
+This solution works fine in most cases, although the feature itself has limited capabilities. For example, it doesn't allow you to overwrite any unwanted changes on the branch, as it only merges the upstream `master` branch.
+
+If you encountered any problems while using the GitHub tool or you want to overwrite your branch's contents, you can use the workflow written by the osu! wiki contributors.
+
+1. Open **your fork** and go to the `Actions` tab.
+2. In `Workflows`, look for `Sync from osu! upstream`.
+3. Click `Run workflow` and fill in the options:
+
+![](img/github-actions-workflow-dialog.png "GitHub Actions Workflow - Run Workflow")
+
+- **Use workflow from**: name of the branch you want to sync. By default, it is set to `master`.
+- **Overwrite any changes in the target repository**:
+  - `true`: replace the contents of your branch with a clean copy of the `master` branch from `ppy/osu-wiki`.
+  - `false` (default): merge your changes with the ones on `ppy/osu-wiki`.
+- **Create a backup of your target branch**:
+  - `true`:  make a branch called `backup-{name of your branch}` before changing it.
+  - `false` (default): do not make any backups.
+
+4. Click the `Run Workflow` button and wait for the workflow to complete. If you're curious about how the tool works, click on the `Sync from osu! upstream` workflow task.
+
+![](img/github-actions-workflow-overview.png "GitHub Actions Workflow - Workflow Overview")
+
+## Making edits
+
+Within your fork of the osu! wiki, you are free to make any changes and save them. **Commits** are individual "save points" of the repository. **Branches** are workspaces, which let you switch between multiple versions of the repository. To make your workflow easier and keep the history of the wiki clean and free from noise, follow these guidelines:
+
+- [Sync the `master` branch](#syncing-the-fork).
+- Always start the work by creating a new branch off `master`, and only keep your changes in there. Give it a meaningful name, such as `update-staff-log`.<!-- https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow is the explanation, but it doesn't really fit in here -->
+- Commit your work when you've made reasonably sized changes. It's better to commit an article as a whole rather than 10 small edits.
```

Comment:
we should probably mention ways in which you can commit multiple times and then squash them together before pull requesting